cellengine/resources/gate.py

```diff
@@ -193,51 +96,177 @@ def create(
         x2: float,
         y1: float,
         y2: float,
-        label: List[str] = [],
+        label: List[float] = [],
         gid: str = None,
         locked: bool = False,
-        parent_population_id: str = None,
-        parent_population: str = None,
         tailored_per_file: bool = False,
         fcs_file_id: str = None,
         fcs_file: str = None,
```

Comment:
Consistency with other gate subclasses: need `Optional[]` on the args that accept `None`.

Please make sure all of the gate subclasses are consistent.